src/WAV.jl

```diff
@@ -6,11 +6,13 @@ export WAVChunk, WAVMarker, wav_cue_read, wav_cue_write, wav_info_write, wav_inf
 export WAVArray, WAVFormatExtension, WAVFormat
 export isextensible, isformat, bits_per_sample
 export WAVE_FORMAT_PCM, WAVE_FORMAT_IEEE_FLOAT, WAVE_FORMAT_ALAW, WAVE_FORMAT_MULAW
+using Compat: codeunits, findall, nothing, Nothing, undef
+import Compat: Libdl
 using FileIO
 
 function __init__()
     module_dir = dirname(@__FILE__)
-    if Libdl.find_library(["libpulse-simple"]) != ""
+    if Libdl.find_library(["libpulse-simple.so.0"]) != ""
```

Comment:
Are there systems where this does not work? I.e. should this be `Libdl.find_library(["libpulse-simple.so.0", "libpulse-simple.so"])` or similar?

Reply:
I don’t think that this works on Mac OS X, where the shared object extension is dylib.

Reply:
Ah, I wasn't aware this was used on OSX, too. Does it still work unversioned there? Or would it have to be `"libpulse-simple.0.dylib"` there? Anyway, I can definitely put the unversioned entry back, than at least it won't break anything.

Reply:
I have not tested recently, but the the bare string used to work on Linux and OS X. I expect it to work on Windows as well, but I have never tested on that operating system.

Reply:
The unversioned string fails on Linux since https://github.com/JuliaLang/julia/pull/22828 (0.7.0-DEV.3588).